1주차/7568.py

```diff
@@ -0,0 +1,26 @@
+#7568
+#덩치가 크다 -> 몸무게, 키 다 커야함
+
+n=int(input())
+D={}
+
+for i in range(n):
+  weight, height = map(int,input().split())
+  D[i]=[weight, height]
+
+lst=[]
+
+for i in range(n):
+  count=1
+  for j in range(n):
+    if list(D[i])[0] < list(D[j])[0] and list(D[i])[1] < list(D[j])[1]:
```

Comment:
```suggestion
    if D[i][0] < D[j][0] and D[i][1] < D[j][1]:
```
D[i]에는 위에서 [weight, height]라는 리스트를 넣어줬기 때문에 굳이 다시 list(D[i]) 같은 형태로 사용하지 않아도 됩니다